src/env.h

```diff
@@ -64,6 +56,18 @@ namespace node {
 #define NODE_PUSH_VAL_TO_ARRAY_MAX 8
 #endif
 
+// PER_ISOLATE_* macros: we're going slightly crazy with macros here but the
+// end hopefully justifies the means. We have a lot of per-isolate properties
+// and adding and maintaining their getters and setters by hand would be
+// a nightmare so let's make the preprocessor generate them for us.
+//
+// In each macro, `V` is expected to be the name of a C function which accepts
+// the number of arguments provided in each tuple in the macro body, typically
+// two. The named function will be invoked against each tuple.
```

Comment:
V is (almost?) always another macro, not a C function. :) (You can change `s/invoked/expanded` for that in the last sentence, if you don’t have a better word in mind.)